wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift

```diff
@@ -228,15 +219,47 @@ final class ClientListViewController: UIViewController,
             successEnrollmentViewController.presentTopmost()
         }
         selectedDeviceInfoViewModel = viewModel
+
         let detailsView = DeviceDetailsView(viewModel: viewModel) {
             self.navigationController?.setNavigationBarHidden(false, animated: false)
         }
         let hostingViewController = UIHostingController(rootView: detailsView)
         hostingViewController.view.backgroundColor = SemanticColors.View.backgroundDefault
+
         navigationController.pushViewController(hostingViewController, animated: true)
         navigationController.isNavigationBarHidden = true
     }
 
+    private func makeDeviceInfoViewModel(
```

Comment:
**suggestion:** Do you think making this into an init method makes more sense?

Reply:
Thanks, but I think it's okay the way it is. If one starts to make it more generic / abstract another level it looks simple in the beginning, but small details like `isFromConversation: false` needs to be configurable and in the end there no real benefit.

This case, I just removed, is the best example:
<img width="862" alt="Screenshot 2024-03-13 at 14 10 15" src="https://github.com/wireapp/wire-ios/assets/7196958/b77103ce-5266-4a87-ab9e-391bc1e8b4b5">